src/utils/helpers/focus-trap/focus-trap.js

```diff
@@ -0,0 +1,49 @@
+const blockTabbing = (ev) => {
+  if (ev.keyCode === 9 || ev.keyCode === 16) {
+    ev.preventDefault();
+  }
+};
+
+const tabbing = (ev, firstFocusableElement, lastFocusableElement) => {
+  if (ev.key === 'Tab' || ev.keyCode === 9) {
+    if (ev.shiftKey) /* shift + tab */ {
+      if (document.activeElement === firstFocusableElement) {
+        lastFocusableElement.focus();
+        ev.preventDefault();
+      }
+    } else if (document.activeElement === lastFocusableElement) {
+      firstFocusableElement.focus();
+      ev.preventDefault();
+    }
+  }
+};
+
+const setFocusTrap = (element) => {
+  // eslint-disable-next-line max-len
+  const ALL_FOCUSABLE_ELEMENTS = 'button, [href], input:not([type="hidden"]), select, textarea, [tabindex]:not([tabindex="-1"])';
+  const focusableElements = element.querySelectorAll(ALL_FOCUSABLE_ELEMENTS);
+
+  if (focusableElements.length <= 0) {
+    document.addEventListener('keydown', blockTabbing);
```

Comment:
What is the purpose of blocking tabbing on the document?

Reply:
If there was a dialog without focusable content you could press the tab key and the focus move out of the dialog. So you could not press escape to close the dialog.